R/lehdr.R

```diff
@@ -31,14 +31,17 @@
 #' @importFrom httr GET stop_for_status HEAD write_disk
 #' @importFrom glue glue
 #' @importFrom stats na.omit
+#' @importFrom stringr str_sub
 #'  
 #' @export
 
-grab_lodes <- function(state, year, lodes_type = c("od", "rac", "wac"), 
```

Comment:
lodes_types can never be a list of all three of these?

Reply:
No. The tables are all separate on the census FTP and folks rarely would want all three. I think it's fine to leave this as is for simplicity sake. We already allow folks to query multiple states and years at the same time.